command/issue.go

```diff
@@ -69,7 +69,10 @@ var issueCreateCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create a new issue",
 	Args:  cmdutil.NoArgsQuoteReminder,
-	RunE:  issueCreate,
+	RunE: func(c *cobra.Command, args []string) error {
+		templateHandler := githubtemplate.GitHubTemplateHandler
+		return issueCreate(c, args, &templateHandler)
+	},
```

Comment:
IMO, We can use this trick to stub anything.  
No need to create extra command-specific structs, and build them here and pass them to the command, to be read there.  
All that is internally handled by cobra here.